foreman/foreman.spec

```diff
@@ -219,6 +219,8 @@ BuildRequires: npm(babel-preset-react) < 7.0.0
 BuildRequires: npm(babel-preset-react) >= 6.5.0
 BuildRequires: npm(c3) >= 0.4.11
 BuildRequires: npm(c3) < 1.0.0
+BuildRequires: npm(compression-webpack-plugin) >= 0.3.2
```

Comment:
Looks like these changes have to go on a different PR to ensure this gets built after the deps.